CHANGES.md

```diff
@@ -2,6 +2,46 @@
 
 ### Enhancements
 
+* Included support for Azure Blob Storage filesystem by adding `AzureFsAccessor()` with support 
+  for the 'abfs' protocol (fsspec adlfs package) class in `/xcube/core/store/fs/impl/fs.py` 
+  and `/xcube/core/store/fs/registry.py`.
+  
+  These changes will enable access to data cubes (`.zarr` or `.levels`) in Azure blob storage. This
+  has been tested with xcube `new_data_store()` function and xcube server configuration file for DataStores.
+  
+  ```python
+    new_data_store(
+        "abfs", # Azure fs protocol
+        root="blob_container",  # Azure blob container name
+        storage_options= {'anon':True, 'account_name':"xxx", 'account_key':'xxx...'} # or connection_string:xxxxxxxx
+    )
+    ```
+  Configuration file sample for Azure Blob
+  Here you can either use `account_name` and `account_key` or use only the `connection_string` which contain both name and key
+  
+  ```yaml
+  DataStores:
+  - Identifier: siec
+    StoreId: abfs
+    StoreParams:
+      root: zarr_cubes
+      max_depth: 1
+      storage_options:
+        anon: true
+        account_name': "xxx"
+        account_key': 'xxx...' # or
+
+        connection_string: xxxxxxxx
+    Datasets:
+      - Path: "*.levels"
+        Style: default
+
+* Added Notebook
+  [test_AzureBlobFS.ipynb](/test/test_AzureBlobFS.ipynb). This notebook shows how a
+  new data store instance can connect and list zarr files from Azure bolb storage using 
+  'abfs' as store id, blob container as root, account_name and account_key or connection_string as store
+  params. 
```

Comment:
See below: Please adapt your Notebook to the conventions used in `example/notebooks/datastores` and move it there. Use the same naming convention.

Mention its new location here.